cmake/packaging/python.dist-info/INSTALLER

```diff
@@ -0,0 +1 @@
+cmake
```

Comment:
[Consider][minor] I wonder if a README file for the .dist-info dir, (or maybe the packaging dir?) might be helpful. I imagine it would be difficult to understand what these files are for when coming across them. And a simple "These files are templates for generating .dist-info when creating a python package via cmake" in a readme or something could help.

I understand a README file might seem a bit overkill here ... I wish directories themselves had metadata you could add to them which could serve as comments, or something like that.

Reply:
Added a README in 2c654bfd (for first commit that creates the directory) then updated the text in c718188f (as part of adding the dist-info).